styles.css

```diff
@@ -0,0 +1,140 @@
+  flex: 4;
+  align-self: center;
+}
+
+.Mobile {
+  height: 100vh;
+  background: url('/photos/Ellipse\ 3\ \(1\).png') 100% 5% no-repeat, url('/photos/Ellipse\ 12.png') 5% 11% no-repeat, url('/photos/Ellipse\ 4.png') 50% 98% no-repeat, url('/photos/Rectangle\ 4.png') 90% 0% no-repeat, url('/photos/Ellipse\ 12.png') 2% 80% no-repeat, url('/photos/Rectangle\ 5.png') 600px 500px no-repeat, url('/photos/Ellipse\ 13.png') 2% 85% no-repeat, url('/photos/Ellipse\ 6.png') 85% 70% no-repeat, url('/photos/Ellipse\ 5.png') 50% 15% no-repeat, url('/photos/Rectangle\ 7.png') 50% 12.3% no-repeat, url('/photos/Rectangle\ 6.png') 0% 5% no-repeat;
+}
+
+a {
+  text-decoration: none;
+}
+
+@media all and (max-width: 375px) {
+  .Mobile {
+    height: 718px;
+    background-image:
+      url('/photos/Ellipse\ 3\ \(1\).png'),
+      url('/photos/Ellipse\ 12.png'),
+      url('/photos/Ellipse\ 4.png'),
+      url('/photos/Rectangle\ 4.png'),
+      url('/photos/Ellipse\ 12.png'),
+      url('/photos/Ellipse\ 13.png'),
+      url('/photos/Ellipse\ 13.png'),
+      url('/photos/Ellipse\ 5.png'),
+      url('/photos/Rectangle\ 7.png'),
+      url('/photos/Rectangle\ 6.png');
+    background-position:
+      100% 5%,
+      5% 11%,
+      50% 98%,
+      90% 0.2%,
+      2% 80%,
+      600px 500px,
+      2% 85%,
+      50% 15%,
+      50% 12.3%,
+      1% 5%;
+    background-repeat:
+      no-repeat,
+      no-repeat,
+      no-repeat,
+      no-repeat,
+      no-repeat,
+      no-repeat,
+      no-repeat,
+      no-repeat,
+      no-repeat,
+      no-repeat;
+  }
+
+  .header {
+    display: flex;
+    list-style: none;
+    justify-content: flex-end;
+  }
+
+  li {
+    width: 68px;
+    text-decoration: none;
+  }
+
+  .first {
+    flex: 4;
+    width: 76px;
+    height: 20px;
+    padding-top: 10px;
+    padding-left: 12px;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    font-family: Arial, Helvetica, sans-serif;
+  }
+
+  .title {
+    width: 327px;
+    height: 1px;
+    padding-top: 150px;
+    padding-left: 24px;
+    padding-right: 24px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: rgba(23, 43, 77, 1);
+    position: static;
+    display: flex;
+    align-items: center;
+    flex: none;
+    margin: 12px;
+  }
+
+  .Bio {
+    width: 327px;
+    height: 146px;
+    padding-top: 40px;
+    padding-left: 24px;
+    padding-right: 24px;
+    color: rgba(52, 69, 99, 1);
+    font-weight: 400;
+    line-height: 24px;
+    font-size: 16px;
+    position: static;
+    font-family: Arial, Helvetica, sans-serif;
+    margin: 1px;
+  }
+
+  .social {
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    position: static;
+    color: rgba(127, 140, 255, 1);
+    width: auto;
+    height: 24px;
+    margin: 1px 8px;
+  }
+
+  .icon {
+    display: flex;
+  }
+
+  img {
+    margin: 5px;
+  }
+}
```

Comment:
Let's style the web app according to the Figma template you choose(Mobile version)
![image](https://user-images.githubusercontent.com/66112760/164886564-a9696518-aa6d-4143-8827-d9d379b2a7b6.png)

Reply:
@bimbolabuari I am not sure on how to set up the mobile version because i thought using media queries would do that. If there is another way please offer a guide